nbs/docs/tutorials/008_batching.ipynb

```diff
@@ -4,7 +4,16 @@
    "cell_type": "code",
```

Comment:
<p>why do we see tasklist2 running? isnt the input value even?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/stringdale/pull/22/discussion/#comment-2628076142'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>